examples/python/cli/cli.py

```diff
@@ -223,6 +234,9 @@ def main():
                         help='The network the SDK runs on, either "MAINNET" or "TESTNET"',
                         type=parse_network)
     subparser = parser.add_subparsers(title='subcommands')
+    # info
+    info_parser = subparser.add_parser('info', help='Get wallet info')
+    info_parser.set_defaults(run=get_info)
```

Comment:
When testing this I always got a default response, we should wait for the initial `SYNCED` event before requesting the info